calico_rkt/calico_rkt.py

```diff
@@ -219,31 +237,154 @@ def _create_rules(id_):
     rules = Rules.from_json(rules_json)
     return rules
 
-def _apply_rules(profile_name, client):
+
+def _apply_default_rules(profile_name):
     """
-    Generate a new profile with the default "allow all" rules.
+    Generate a new profile rule list and update the _datastore_client
     :param profile_name: The profile to update
     :type profile_name: string
     :return:
     """
     try:
-        profile = client.get_profile(profile_name)
+        profile = _datastore_client.get_profile(profile_name)
     except:
-        print_stderr("Error: Could not apply rules. Profile not found: %s, exiting" % profile_name)
+        _log.error(
+            "Could not apply rules. Profile not found: %s, exiting" % profile_name)
         sys.exit(1)
 
-    profile.rules = _create_rules(profile_name)
-    client.profile_update_rules(profile)
-    print_stderr("Finished applying rules.")
+    profile.rules = _create_default_rules(profile_name)
+    _datastore_client.profile_update_rules(profile)
+    _log.info("Finished applying rules.")
+
 
-def _allocate_IP():
+def _generate_pool(subnet):
```

Comment:
I took a closer look and I think this function should be removed. All it really does is call a datastore function after a very small type conversion. Can you move the functionality up into container_add? Same with  _allocate_ip_pool (But definitely argue for keeping it if you think It'll be useful later on!)